wa-public/src/app/components/credential-issuance/credential-issuance.component.ts

```diff
@@ -121,19 +120,26 @@ export class CredentialIssuanceComponent implements OnInit, OnDestroy {
         switchMap(() => this.actionSvc.getCredentialById(this.credExId)) // TODO: @SH replace with this.transactionStateURL
       )
       .subscribe(res => {
-        console.log(res)
-        if (!res) return
-        this.updateProgress(res.issued)
+        console.log(res);
+        if (!res) return;
+        if (this.step === 3) return this.completeProgress();
+
+        this.updateProgress(res.issued);
       }));
     // setTimeout(() => (this.step = 3), 20000);
     const user = this.stateSvc.user;
     if (user) this.$user = of(`${user.firstName}`);
   }
   ngOnDestroy() {
-    this.subs.forEach(sub => sub.unsubscribe())
+    this.subs.forEach(sub => sub.unsubscribe());
   }
 
   updateProgress(issued: boolean) {
     if (issued) this.step = 3;
   }
+
+  async completeProgress() {
+    await this.actionSvc.logout('https://identity-kit.pathfinder.gov.bc/completed');
+    // this.router.navigate(['/completed']);
```

Comment:
Does the new flow mean the "last check on the issuance page is never displayed, but the uiser is redirected to the completed page instead?

Reply:
issuance is displayed. After a few seconds (when the polling returns an issued status on the second check) then it will re-direct the user to the final page.